app/controllers/api/v1/users_controller.rb

```diff
@@ -110,14 +114,15 @@ def index
     #{json_schema(Api::V1::UserRepresenter, include: :readable)}
   EOS
   def show
+    return head(:forbidden) if current_human_user.is_anonymous?
```

Comment:
I think you're correct. I did this because of failing tests - but I think Dante helped me fix the test config file so it shouldn't need this anymore.